ntp.go

```diff
@@ -145,10 +145,11 @@ func (m *msg) getLeapIndicator() LeapIndicator {
 // QueryOptions contains the list of configurable options that may be used with
 // the QueryWithOptions function.
 type QueryOptions struct {
-	Timeout time.Duration // defaults to 5 seconds
-	Version int           // NTP protocol version, defaults to 4
-	Port    int           // NTP Server port for UDPAddr.Port, defaults to 123
-	TTL     int           // IP TTL to use for outgoing UDP packets, defaults to system default
+	Timeout      time.Duration // defaults to 5 seconds
+	Version      int           // NTP protocol version, defaults to 4
+	LocalAddress string        // IP to use for the client address
```

Comment:
suggested comment:

`// Client local address, nil to choose automatically`

Reply:
Done